src/TextField/TextFieldHint.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Transitions from '../styles/transitions';
+import styleUtils from '../utils/styles';
+
+const propTypes = {
+  /**
+   * True if the hint text should be hidden.
+   */
+  hidden: React.PropTypes.bool,
+
+  /**
+   * The material-ui theme applied to this component.
+   */
+  muiTheme: React.PropTypes.object.isRequired,
+
+  /**
+   * Override the inline-styles of the hint text.
+   */
+  style: React.PropTypes.object,
+
+  /**
+   * The hint text displayed.
+   */
+  text: React.PropTypes.string,
+};
+
+const defaultProps = {
+  visible: true,
```

Comment:
Looks like a typo.
When we have the choice between `hidden`, `show` and `visible`, I would personally go for `show`.
I also think that `show` is what it was chosen in some other component.